modules/apps/product-navigation/product-navigation-taglib/src/main/resources/META-INF/resources/personal_menu/js/PersonalMenu.es.js

```diff
@@ -41,7 +41,31 @@ function PersonalMenu({
 
 	return (
 		<ClayDropDownWithItems
-			items={items}
+			items={items.map((item) => {
+				const {jsOnClickConfig, ...otherKeys} = item;
+
+				return {
+					onClick: () => {
+						const {
+							selectEventName,
+							namespace,
+							url,
+							title,
+						} = jsOnClickConfig;
+
+						Liferay.Util.openSelectionModal({
+							id: namespace + 'selectSite',
```

Comment:
In this case, it looks like this `id` and the `selectEventName` are equivalent. Would it work to just re-use `selectEventName`?